Metadata.Rmd

```diff
@@ -529,8 +529,8 @@ The `person()` function has four main inputs:
 
     * `cre`: the creator or maintainer, the person you should bother 
       if you have problems. Despite being short for "creator", this is the
-      correct role to use for the current maintainer, even if they are not
-      not the initial creator of the package.
+      correct role to use for the current maintainer, even if they are not 
```

Comment:
I just fixed this myself locally. Thanks for catching the extra "not". But I didn't want to merge the PR and gain this trailing whitespace that wasn't there before.